src/main.cpp

```diff
@@ -68,11 +68,11 @@ void setup()
 void setGainsFromRcTrimmers()
 {
   controller.setQuaternionGain(
-      -QUATERNION_XY_GAIN * RADIO_TRIMMER_MAX_QUATERNION_XY_GAIN);
+      -raspberryEsp32Interface.trimmerVRAPercentage() * RADIO_TRIMMER_MAX_QUATERNION_XY_GAIN);
```

Comment:
@jamie-54, this looks good - just write at least a comment with the exact values that seem to work. We should be using these as default values in the future. The code you have here will allow us to fine tune it (make it more or less aggressive in the future). It will also allow us to tune the second quadcopter where the parameters will be a bit different. We may also have to fine-tune again after we mount the anemometer and the GPS antenna.

Here #159 would be convenient: it would allow us to either use the default values for these gains, or use the trimmers to tune them. If we choose to use the default values we can reassign the trimmers to something else (e.g., the altitude control or position control gains).

Reply:
@alphaville I have changed the gain values in the `config.hpp` file (lines 72, 80 and 85) to values that seem to work well. If we change lines 71, 73 and 75 in the `main.cpp` back to the way they were this will use the hard set values.

I agree if #159 will help with this in future